ukvm/ukvm-configure

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: ukvm-configure UKVM_SRC [MODULES]"
+    echo "    UKVM_SRC is /path/to/ukvm"
+    echo "    MODULES can be any combination of: net blk gdb"
+    exit
+fi
+
+UKVM_SRC=`readlink -e $1`
+shift
+UKVM_MODULES=$@
+
+(
```

Comment:
You can simplify this a lot by using make string operations, and possibly not even needing to have a generated Makefile.ukvm. Something like this:

`UKVM_MODULES=blk net
all:
        gcc $(addprefix -DUKVM_MODULE_,${UKVM_MODULES}) $(addprefix _build-ukvm/ukvm-,${UKVM_MODULES})`